asmcli/asmcli.sh

```diff
@@ -1061,7 +1062,7 @@ validate_args() {
 MODE
 EOF
 
-  local CLUSTER_DETAIL_SUPPLIED=0
+  local CLUSTER_DETAIL_SUPPLIED=0 # gke cluster param usage intended
```

Comment:
Since this variable is set to 0 and KUBECONFIG_SUPPLIED is set to 1 below, maybe the comments should be less parallel?  "whether gke cluster params were passed via CLI" and "whether a kubeconfig file was supplied" are clearer to me.

Reply:
done! made the comments consistent.